legacy/src/api/payment-requests/payment-requests.md

```diff
@@ -373,6 +373,8 @@ Payment Activities are created when a Payment Request has been **created**, **pa
 | configId             | String            | The [Merchant Config][] id used to configure the payment options.                                                                                                        |
 | value                | {% dt Monetary %} | The canonical value of the payment request. Must be positive.                                                                                                            |
 | barcode              | String {% opt %}  | The [Scanned Code] to identify the account to attach the payment request to                                                                                              |
+| barcodeType          | String {% opt %}  | An attribute to specify or identify the type of barcode that is being used or processed                                                                                  |
+| collectionId         | String {% opt %}  | The Collection Id                                                                                                                                                        |
```

Comment:
Might be more informative if description mentions tokens, e.g. The identifier of the token collection

Reply:
Thanks for the feedback ⭐️  I will update it 👍